### ISSUE TITLE
[BUG] An interactive customised component does not get relayed as such to screen reader users.

### ISSUE THREAD

**Issue by KatiePUX** [UKHO/admiralty-design-system#207]
**Describe the bug**
A customised component has been used for the process of uploading files. This instance can be found with the step 7 select files section. The component is missing accessible roles and labels, which has affected the ability to interact with the component via assistive technologies.
The component is accessible for keyboard only users navigating the service, but no focus indicator is provided, making it difficult to access the feature. Screen reader users will only be able to upload their file if they press enter on the ‘select file or drag it here’ text only in context, which does not get relayed as a component and is therefore an unexpected interaction.

**Expected behaviour**
Customised components are commonly found to lack accessible functionalities that are present with native HTML elements. The component needs certain roles and other accessible attributes to allow assistive technology users to interact with these customised components.
Customised components should be easily accessible to all users navigating the service, and not only by mouse and keyboard.
We suggest implementing a file upload component similarly to the file upload GOV.UK Design System component.

**Additional context**
See DAC accessibility report page 21 -22
